apps/prairielearn/src/server.js

```diff
@@ -108,6 +108,14 @@ module.exports.initExpress = function () {
     }
   }
 
+  //////////////////////////////////////////////////////////////////////
+  //////////////////////////////////////////////////////////////////////
+  //////////////////////////////////////////////////////////////////////
+  // Webhooks //////////////////////////////////////////////////////////
```

Comment:
```suggestion
  // This should come before the session middleware so that we don't
  // create a session every time we get a health check request.
```